io.github.shadowmario.fnf-psychengine.yml

```diff
@@ -0,0 +1,91 @@
+app-id: io.github.shadowmario.fnf-psychengine
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: io.github.shadowmario.fnf-psychengine.sh
+
+finish-args:
+  # gamepad support
+  - --device=all
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Audio
+  - --socket=pulseaudio
+
+modules:
+  - name: fnf-psychengine
+    only-arches:
+      - x86_64
```

Comment:
so what happen in the aarch64 package?

Reply:
I was assuming the build would be skipped if there aren't any modules... is moving the `only-arches` to the root of the spec the right way to do this?

Reply:
No you need a flathub.json file too https://docs.flathub.org/docs/for-app-authors/maintenance#only-build-on-x86_64